backgrounds.json

```
@@ -472,5 +472,22 @@
             "https://cdn.spacetelescope.org/archives/images/wallpaper4/opo9935d.jpg",
             "https://cdn.spacetelescope.org/archives/images/wallpaper4/opo9828c.jpg"
         ]
+    },
+    "iwm": {
+        "title": "Imperial War Museum Wallpapers",
```

```suggestion
        "title": "Imperial War Museum",
```

The only reason I added "wallpapers" to Hubble was that this wasn't strictly released as virtual background collection, but just general wallpapers. I think it's best to remove "wallpapers" here to keep it short and simple.